Tech@HubAPI/Tech@HubAPI/Controllers/UserController.cs

```diff
@@ -15,5 +16,26 @@ public UserController(DatabaseContext dbContext)
 		{
 			_dbContext = dbContext;
 		}
+		
+		[HttpGet]
+		[Route("get/{ID}")]
+		public User GetUser(int ID) 
```

Comment:
So to fix the syntax error that line 33 causes, change this line (the function signature) to read
```
public ActionResult<User> GetUser(int ID)
```
That should fix the problem